report_thesis/src/sections/results/initial_results.tex

```diff
@@ -47,23 +51,22 @@ \subsection{Initial Results}
 \end{figure*}
 
 \begin{table}[ht]
-\centering
-\begin{tabularx}{\linewidth}{lrr}
+\begin{tabularx}{lrr}
 \toprule
 Model & Relative Performance (\%) & Diff. vs Next (\%) \\
 \midrule
-\gls{xgboost} & 100.00 & 0.85 \\
-\gls{svr} & 100.85 & 2.22 \\
-\gls{gbr} & 103.07 & 0.87 \\
-\gls{ngboost} & 103.94 & 0.51 \\
-\gls{rf} & 104.45 & 0.39 \\
-\gls{etr} & 104.84 & 3.89 \\
-Ridge & 108.74 & 2.92 \\
-\gls{pls} & 111.66 & 2.73 \\
-\gls{lasso} & 114.38 & 13.44 \\
-\gls{ann} & 127.82 & 15.36 \\
-\gls{cnn} & 143.18 & 39.18 \\
-\gls{enet} & 182.36 & - \\
+XGB & 100.00 & - \\
+SVR & 100.85 & 0.85 \\
+GBR & 103.07 & 2.22 \\
+NGB & 103.94 & 0.87 \\
+RandomForest & 104.45 & 0.51 \\
+ExtraTrees & 104.84 & 0.39 \\
+Ridge & 105.04 & 0.20 \\
+PLS & 111.66 & 6.61 \\
+ElasticNet & 114.12 & 2.46 \\
+LASSO & 114.30 & 0.19 \\
+ANN & 127.82 & 13.52 \\
+CNN & 143.18 & 15.36 \\
 \bottomrule
 \end{tabularx}
 \caption{Relative performance of each model compared to the best performing model, and the difference in performance compared to the next best model.}
```

Comment:
A few points on this one:
- Diff vs Prev. (%) is misleading because it's measured in percentage points / actual values, not percentages
- Need to clarify performance is measured by RMSECV, and that a higher percentage is worse
- Might need text clarifying what's shown ('normalized' RMSECV * 100 for percentage, with the best performing as the 'baseline'). But it's kinda covered in the 'assigned a relative performance of 100%'

Reply:
```suggestion
\caption{Relative performance of each model compared to the best performing model, measured by normalized RMSECV (multiplied by 100 for percentage). A higher percentage indicates worse performance. The 'Diff. vs Prev.' column shows the difference in performance compared to the next best model, measured in percentage points.}
```

Reply:
> Diff vs Prev. (%) is misleading because it's measured in percentage points / actual values, not percentages

I can't really think of better column names that would encapsulate this without being far too long. It barely fits as is. Any ideas?

Reply:
Drop `(%)`  and just have `Diff vs. Prev.`?